src/frontend/components/StudentLiveControlButtons/StudentHideChatButton/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+
+import { Button } from 'components/Button';
+import { ChatSVG } from 'components/SVGIcons/ChatSVG';
+import { useLivePanelState } from 'data/stores/useLivePanelState';
+
+const messages = defineMessages({
+  chat: {
+    defaultMessage: 'Chat',
+    description: 'Label for the chat button',
+    id: 'components.StudentHideChatButton.Chat',
+  },
+  hideChat: {
+    defaultMessage: 'Hide chat',
+    description: 'Title for the chat button',
+    id: 'components.StudentHideChatButton.hideChat',
+  },
+});
+
+export const StudentHideChatButton = () => {
+  const intl = useIntl();
+  const { openPanel } = useLivePanelState((state) => ({
+    openPanel: state.setPanelVisibility,
```

Comment:
IMO you should keep `setPanelVisibility` instead of `openPanel`. `openPanel` is an action and I didn't expect to call it with a boolean as argument

Reply:
https://github.com/openfun/marsha/pull/1249/commits/815336b1016f6d911ec51c3d5d04074e8808984c